addons/l10n_pe/demo/account_demo.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+from odoo import api, models
+
+
+class AccountChartTemplate(models.Model):
+    _inherit = "account.chart.template"
+
+    @api.model
+    def _get_demo_data_move(self):
+        cid = self.env.company.id
+        model, data = super()._get_demo_data_move()
+        document_type = self.env.ref('l10n_pe.document_type01')
+        data_dict = dict(data)
+        data_dict[f'{cid}_demo_invoice_1']['l10n_latam_document_type_id'] = document_type.id
+        data_dict[f'{cid}_demo_invoice_2']['l10n_latam_document_type_id'] = document_type.id
+        data_dict[f'{cid}_demo_invoice_3']['l10n_latam_document_type_id'] = document_type.id
+        data_dict[f'{cid}_demo_invoice_followup']['l10n_latam_document_type_id'] = document_type.id
+        data_dict[f'{cid}_demo_invoice_5']['l10n_latam_document_number'] = '1'
+        data_dict[f'{cid}_demo_invoice_equipment_purchase']['l10n_latam_document_number'] = '2'
+        return model, data
```

Comment:
What's the point of shallowly copying the top-level dict, updating subkeys (so the copy does nothing) then not even return the copy? It's useless in every override.

Reply:
Because it is a shallow copy, it updates the `data`, which I'm returning.

Reply:
> Because it is a shallow copy

Which makes it useless.

>  it updates the data, which I'm returning

Well yes, you're updating `data` in-place through the copy, so why bother with a copy? It doesn't do anything useful.

Reply:
What do you propose instead? It is more readable to me.

Reply:
> What do you propose instead? 

To not copy anything? That literally has no effect other than paying for an unnecessary copy.

> It is more readable to me.

How is
```python
data_dict = dict(copy)
data_dict['foo']['baz'] = bar
```
more readable than just
```python
data['foo']['baz'] = bar
```
?

Reply:
`data` is a list of  tuples.

Reply:
Makes sense, then. Thanks.

Why not make it a dict though? It clearly has to fit in one.

Reply:
it's ok like that for me

Reply:
I changed the declaration to a dict. It was needed on a previous implementation but it works now thanks to the fact that `_get_demo_data` is a generator now.